locales/en-US/messages.properties

```diff
@@ -15,10 +15,13 @@ newProjectBtn=Start a project from scratch
 contactEmail=Email Us
 contactTwitter=Talk to us on Twitter
 readTheDocs=Read the Docs
-contribute=Contribute on Github
+contribute=Contribute on GitHub
```

Comment:
So does that mean it's OK to leave it as it is @TheoChevalier - or should I change the string ID?